src/app/modules/group/pages/joined-groups/joined-groups.component.html

```diff
@@ -4,3 +4,9 @@
     <span class="group-subtitle">{{ subtitle }}</span>
   </div>
 </div>
+
```

Comment:
Ok, please fix (in another PR ?) the subtitle. Actually put the title and subtitle in template, I find it confusing to have these in variable anyway. And actually the font does not seem really correct either.

Reply:
fixed